validate/builders.go

```diff
@@ -251,7 +263,7 @@ func GT(right interface{}) Validator {
 			return nil
 		}
 
-		return NewError(fmt.Sprintf("%v is not greater than %v", left, right))
+		return NewError(fmt.Sprintf("property is not greater than %v", right))
```

Comment:
@abrykajlo I think it's better an imperative message, like "property should be greater than", for all messages.